### PR TITLE
Dev instructions

### PR DESCRIPTION
**Pull request by edsu**
This PR adds a few instructions for getting a working dev environment going. Please disregard if they aren't helpful!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -38,3 +38,5 @@ yarn.lock
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+
+.node-version
```
Comment (mjgiarlo) [inline .gitignore:42]: You may want to consider adding this to your global `.gitignore`, too (or instead?)

**Review by mjgiarlo — APPROVED** [review]
LGTM! We should prolly defer to @sul-dlss/access-team folk on merging.

File: .gitignore (hunk 1 above)
Comment (edsu) [reply .gitignore]: Good point, I just removed it.

File: README.md [hunk 2]
```diff
@@ -8,23 +8,30 @@ An [oEmbed](http://oembed.com/) provider for embedding resources from the Stanfo
...
 
-To install needed JavaScript dependencies make sure to install them using:
+## oEmbed specification details
+
+URL scheme: `http://purl.stanford.edu/*`
 
-```sh
-$ yarn install
-```
+API endpoint: `TBD`
```
Comment (thatbudakguy) [inline README.md:32]: what's it mean that this is TBD?

**Review by thatbudakguy — APPROVED** [review]
Thanks for adding this!

File: README.md (hunk 2 above)
Comment (thatbudakguy) [reply README.md:32]: ...I guess that was already there. I think this should be `embed.stanford.edu` now, probably?

Comment (mjgiarlo) [reply README.md:32]: I guess?

Comment (mjgiarlo) [reply README.md:32]: Either that or we can consider renaming the prod embed service `TBD.stanford.edu`!

Comment (edsu) [reply README.md]: This was out of the scope of the changes I was making to just make it a bit easier to get up and running for development. But I just pushed a change to make this hostname and the example to hopefully reflect reality.

**Pull request merged**